elastic-agent-shipper.spec.yml

```
@@ -1,14 +1,21 @@
 version: 2
-outputs:
-  - name: elasticsearch
-    description: Elasticsearch
-    platforms: &platforms
+shippers:
+  - name: shipper
+    description: "Elastic Agent Shipper"
+    platforms:
       - linux/amd64
       - linux/arm64
       - darwin/amd64
       - darwin/arm64
       - windows/amd64
       - container/amd64
       - container/arm64
-    command: {}
-  
\ No newline at end of file
+    outputs:
+      - elasticsearch
+      - kafka
+      - logstash
+      - redis
+    command: &command
+      args:
+        - "-E"
+        - "logging.level=debug"
```

```suggestion
        - "logging.level=info"
```

The agent wants us to start at the info level, since that is the level the agent itself starts at.